Cannot install @nx-tools/nx-container due to conflicting tslib@2.5.0 dependency

### Current Behavior

When trying to install @nx-tools/nx-container installation fails due to conflicting dependency.

### Expected Behavior

Installation should succeed.

### Steps to Reproduce

1. npm install @nxlv/python --save-dev
2. npm install -D @nx-tools/nx-container

### Nx Report

```shell
Node   : 20.8.0
   OS     : linux-x64
   npm    : 10.1.0
   
   nx             : 16.10.0
   @nx/js         : 16.10.0
   @nx/workspace  : 16.10.0
   @nx/devkit     : 16.10.0
   @nrwl/tao      : 16.10.0
   typescript     : 5.2.2
   ---------------------------------------
   Community plugins:
   @nx-tools/nx-container : 5.0.3
   @nxlv/python           : 16.1.2
```


### Failure Logs

```shell
~/R/my-package (main|✚1) $ npm install -D @nx-tools/nx-container
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: my-package@0.0.0
npm ERR! Found: tslib@2.5.0
npm ERR! node_modules/tslib
npm ERR!   tslib@"^2.3.0" from @nrwl/tao@16.10.0
npm ERR!   node_modules/@nrwl/tao
npm ERR!     @nrwl/tao@"16.10.0" from nx@16.10.0
npm ERR!     node_modules/nx
npm ERR!       nx@"16.10.0" from @nrwl/tao@16.10.0
npm ERR!       3 more (@nx/devkit, @nx/workspace, the root project)
npm ERR!   tslib@"^2.3.0" from @nx/devkit@16.10.0
npm ERR!   node_modules/@nx/devkit
npm ERR!     @nx/devkit@"16.10.0" from @nrwl/devkit@16.10.0
npm ERR!     node_modules/@nrwl/devkit
npm ERR!       @nrwl/devkit@"16.10.0" from @nx/devkit@16.10.0
npm ERR!     @nx/devkit@"16.10.0" from @nx/js@16.10.0
npm ERR!     node_modules/@nx/js
npm ERR!       @nx/js@"16.10.0" from @nrwl/js@16.10.0
npm ERR!       node_modules/@nrwl/js
npm ERR!         @nrwl/js@"16.10.0" from @nx/js@16.10.0
npm ERR!       1 more (the root project)
npm ERR!     2 more (@nx/workspace, @nxlv/python)
npm ERR!   6 more (@nx/js, @nx/workspace, @nxlv/python, @yarnpkg/parsers, nx, rxjs)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! dev @nx-tools/nx-container@"*" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: tslib@2.6.2
npm ERR! node_modules/tslib
npm ERR!   peer tslib@"^2.5.3" from @nx-tools/nx-container@5.0.3
npm ERR!   node_modules/@nx-tools/nx-container
npm ERR!     dev @nx-tools/nx-container@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```


### Additional Information

_No response_

--- a/packages/nx-python/package.json
+++ b/packages/nx-python/package.json
@@ -14,6 +14,9 @@
   "peerDependencies": {
     "@nx/devkit": "^16.0.0"
   },
+  "dependencies": {
+    "tslib": "^2.3.0"
+  },
   "nx-migrations": {
     "migrations": "./migrations.json"
   }

